.cloudbuild.yaml

```diff
@@ -56,14 +56,14 @@ steps:
 - name: 'gcr.io/cloud-builders/docker'
   args: ['build', '-t', 'gcr.io/$PROJECT_ID/frontend:$COMMIT_SHA',
          '--build-arg', 'COMMIT_HASH=$COMMIT_SHA',
-         '--build-arg', 'TAG_NAME=$TAG_NAME',
+         '--build-arg', 'TAG_NAME=$(cat /workspace/VERSION)',
```

Comment:
Are you sue we're not overwriting the released tag on every master commit?

Reply:
this is not up-to-date in fact

Reply:
also it's kind of weird this shows up in the diff, this change was cherry picked from master.